pages/mobile/login.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.desktop.base import Base
+from selenium.webdriver.common.by import By
+
+
+class Login(Base):
+
+    _page_title = 'Sign In | Mozilla Support'
+    _page_url = '/en-US/users/login'
+    _username_box_locator = (By.ID, 'id_username')
+    _password_box_locator = (By.ID, 'id_password')
+    _log_in_button_locator = (By.CSS_SELECTOR, "button[data-name='login']")
+    _login_error_locator = (By.CSS_SELECTOR, 'ul.errorlist > li')
+
+    # if user is logged-in then you see these elements
+    _sign_out_locator = (By.ID, '.sign-out')
+
+    def log_in(self, username, password):
+        self.selenium.find_element(*self._username_box_locator).send_keys(username)
+        self.selenium.find_element(*self._password_box_locator).send_keys(password)
+        self.selenium.find_element(*self._log_in_button_locator).click()
+
+        if not self.is_user_logged_in:
+            error = self.selenium.find_element(*self._login_error_locator).text
+            error = "login failed for %s\n" % username + error
+            Assert.fail(error)
+
+    def is_user_logged_in(self):
+        self.open_menu()
+        return self.selenium.find_element(*self._sign_out_locator)
```

Comment:
This appears to replace `is_user_logged_in` from `home.py`. Is there a difference? Could we put this into `base.py` instead? I see that this opens the menu and returns the element, but it doesn't return `False` if the element is not present.